packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -272,14 +272,16 @@ export class SummaryGenerator {
                         generateTelemetryProps = {
                             ...generateTelemetryProps,
                             clientSequenceNumber: summaryData.clientSequenceNumber,
-                            submitOpDuration: summaryData.submitOpDuration,
                         };
                     }
                 }
             }
 
-            logger.sendTelemetryEvent({ eventName: "GenerateSummary", ...generateTelemetryProps });
-            if (summaryData.stage !== "submit") {
+            if (summaryData.stage === "submit") {
+                logger.sendTelemetryEvent({ eventName: "GenerateSummary", ...generateTelemetryProps });
+            }
+            else {
```

Comment:
#nit This formatting is kind of inconsistent. What do you think about `} else {` ?